SQL/database_changelog.txt

```diff
@@ -8,6 +8,13 @@ INSERT INTO `SS13_schema_revision` (`major`, `minor`) VALUES (5, 2);
 
```

Comment:
Annoyingly I can't actually suggest an edit for the text just above this point. The Minor version needs to be updated to 5.3.

```sql
The latest database version is 5.3; The query to update the schema revision table is:
INSERT INTO `schema_revision` (`major`, `minor`) VALUES (5, 3);
or
INSERT INTO `SS13_schema_revision` (`major`, `minor`) VALUES (5, 3);
```